android/src/main/java/me/shashwatmishra/root_tester/RootTesterPlugin.java

```diff
@@ -64,6 +64,11 @@ public void onDetachedFromEngine(@NonNull FlutterPluginBinding binding) {
   @Override
   public void onMethodCall(@NonNull MethodCall call, @NonNull Result result) {
     if (call.method.equals("isDeviceRooted")) {
+      if (isEmulator()) {
```

Comment:
I think a new method should be there for just checking if it is an emulator or not. would appreciate if you could make one and expose dart API for that also.

Reply:
Hi,

do you prefer to expose the `isEmulator` method to dart API and remove the check inside `isDeviceRooted` method? So developer should do the check manually?

Reply:
yeah, I guess that will give more flexibility to the developers, to only block it for rooted devices & enable emulators during development & profiling mode.

Reply:
two methods, one to check isRooted & one to check if it's a emulator, would be a good idea i guess, what do you sugegst?